api/requirements_new.txt

```diff
@@ -0,0 +1,66 @@
+alembic==1.3.3
```

Comment:
Can we not replace the current requirements file?

Reply:
Yes sure we can.